citizen/citizen.c

```diff
@@ -217,52 +235,67 @@ ResponseCode removeCitizen(List *list, int citizenId) {
 }
 
 // update a citizen's information in the list.
-ResponseCode updateCitizen(List *list, const Citizen *citizen) {
-    if (list == NULL || citizen == NULL) return Failed;
+ResponseCode updateCitizen(List *list, const Citizen *citizen) 
+{
+    if (list == NULL || citizen == NULL) 
+    {
+    return Failed;
+    }
 
     Node *current = list->head;
     int found = 0;
 
     // Traverse the list to find and update the citizen
-    while (current != NULL) {
-        if (current->citizen.citizenId == citizen->citizenId) {
+    while (current != NULL) 
+    {
+        if (current->citizen.citizenId == citizen->citizenId) 
+        {
             found = 1;
             current->citizen = *citizen;
             break;
         }
         current = current->next;
     }
 
-    if (found) {
+    if (found) 
+    {
         return Success;
-    } else {
+    } 
+    else 
+    {
         return NotFound;
     }
 }
 
 // sort a list of citizens based on a specified attribute.
-List sortCitizen(List *list, Type sort) {
+List sortCitizen(List *list, Type sort) 
+{
     List sortedList = {NULL, NULL};
```

Comment:
please follow the naming convention on variables, use the snake_case

Reply:
Done. Thanks.